regression/test.pl

```diff
@@ -71,7 +71,9 @@ ($$$$$)
 
   $options =~ s/$ign//g if(defined($ign));
 
-  my $output = $input;
+  my $descriptor = $test;
+  $descriptor =~ s/^.*\///;
```

Comment:
... or `use File::Basename;` and do `my $descriptor = basename($test);` and have self-explanatory code.